src/TreeView/TreeView.tsx

```diff
@@ -429,24 +439,37 @@ function useSubTree(children: React.ReactNode) {
 
 export type TreeViewVisualProps = {
   children: React.ReactNode | ((props: {isExpanded: boolean}) => React.ReactNode)
+  // Provide an accessible name for the visual. This should provide information
+  // about what the visual indicates or represents
+  label?: string
```

Comment:
Can you add this to the props table for LeadingVisual and TrailingVisual in the docs?

Reply:
Done!